<launch>
  <!-- Parameter -->
  <arg name="config_file" default="$(find-pkg-share autoware_steer_offset_estimator)/config/steer_offset_estimator.param.yaml"/>
  <!-- get wheel base from vehicle info -->
  <include file="$(find-pkg-share autoware_global_parameter_loader)/launch/global_params.launch.py"/>

  <!-- steer offset estimator -->
  <node pkg="autoware_steer_offset_estimator" exec="steer_offset_estimator_node" name="steer_offset_estimator" output="screen">
    <param from="$(var config_file)"/>
    <remap from="~/input/pose" to="/localization/pose_estimator/pose"/>
    <remap from="~/input/steer" to="/vehicle/status/steering_status"/>
    <remap from="~/output/steering_offset" to="~/steering_offset"/>
    <remap from="~/output/steering_offset_covariance" to="~/steering_offset_covariance"/>
    <remap from="~/output/debug_info" to="~/debug_info"/>
  </node>
</launch>
